mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -16,6 +15,16 @@ def _check_playlist_name(name):
         raise exceptions.MpdInvalidPlaylistName()
 
 
+def _get_playlist(context, name, must_exist=True):
+    playlist = None
+    uri = context.lookup_playlist_uri_from_name(name)
+    if uri:
+        playlist = context.core.playlists.lookup(uri).get()
```

Comment:
I'm not doing the full track lookups in here anymore, these are just refs since other than for `listplaylistinfo` that's all we ever want.